app/models/scripts/dataCreation.js

```diff
@@ -1,3 +1,50 @@
+const bcrypt = require('bcrypt');
+
 exports.execute = db => {
-  // This function should create data for testing and return a promise
+  return bcrypt
+    .hash('falabella2017', 10)
+    .then(hash => {
+      const inserts = [];
+
+      inserts.push(
+        db.models.user.create({
+          firstName: 'persona1',
+          lastName: 'apellido1',
+          email: 'email1@wolox.com',
+          password: hash
+        })
+      );
+
+      inserts.push(
+        db.models.user.create({
+          firstName: 'persona2',
+          lastName: 'apellido2',
+          email: 'email2@wolox.com',
+          password: hash
+        })
+      );
+
+      inserts.push(
+        db.models.user.create({
+          firstName: 'persona3',
+          lastName: 'apellido3',
+          email: 'email3@wolox.com',
+          password: hash
+        })
+      );
+
+      inserts.push(
+        db.models.user.create({
+          firstName: 'persona4',
```

Comment:
You can use https://github.com/marak/Faker.js/ here, most of the time is great to have random data here (maybe you can keep the email and the password, and just use this in the name